modules/manage/pages/schema-reg/schema-reg-authorization.adoc

```diff
@@ -197,11 +197,30 @@ For additional guidance on these operations, see the link:/api/doc/schema-regist
 Before you can enable Schema Registry Authorization, you must have:
 
 ifndef::env-cloud[]
-* A valid Redpanda Enterprise license
+* A valid Redpanda Enterprise license.
 endif::[]
 
-* `rpk` v25.2+
-* Cluster administrator permissions to modify cluster configuration
+ifdef::env-cloud[]
+* `rpk` v25.2+ installed. For installation instructions, see xref:manage:rpk/rpk-install.adoc[rpk installation].
+endif::[]
+
+ifndef::env-cloud[]
+* `rpk` v25.2+ installed. For installation instructions, see xref:get-started/rpk-install.adoc[rpk installation].
+endif::[]
+
+* Authentication enabled using `schema_registry_api.authn_method`, which specifies how clients must authenticate when accessing the Schema Registry API. See xref:reference:properties/broker-properties.adoc#schema-registry[Schema Registry broker properties].
```

Comment:
From my testing this is not needed in Cloud when enabling schema registry ACLs. 

But please, @c4milo can you help me confirm what's the default configuration of our schema_registry_api listener in cloud? does it set the authentication method?

Reply:
Discussed with @mattschumpert . Conditionalizing this out for cloud.